2021/1xxx/CVE-2021-1888.json

```diff
@@ -1,18 +1,69 @@
 {
```

Comment:
('Invalid control character at: line 43 column 24 (char 2815)',)

Reply:
Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues